php/class-fieldmanager-button.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Class file for Fieldmanager_Button
+ *
+ * @package Fieldmanager
+ */
+
+/**
+ * Button Field
```

Comment:
```suggestion
 * Button Field.
```

This should end in a full stop